### ISSUE TITLE
Does not work with helm v3.5.2

### ISSUE THREAD

**Issue by yacinelazaar** [datreeio/helm-datree#6]
```
helm datree test . --values values.yaml        
Error: plugin "datree" exited with error
```
```
datree version  
0.1.708
```
```
helm version                           
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"clean", GoVersion:"go1.15.7"}

```

**Comment by eyarz** [comment]
Thank you for reporting that!

We deployed a fix for that so can you please install the latest version and try again?
`helm plugin install https://github.com/datreeio/helm-datree`

**Comment by eyarz** [comment]
Fixed (#11)
